installInstructions/install3rdPartyPackages.rst

```diff
@@ -445,38 +445,47 @@ It is installed with:
 
    pip install petsc4py==<version> --user --no-cache
 
-Advanced install (For multiple PETSc architectures)
-***************************************************
+Build from source (Required for multiple PETSc architectures)
+*************************************************************
 .. WARNING::
    You must compile a unique petsc4py install for each PETSc architecture.
 
-`Download <https://bitbucket.org/petsc/petsc4py/downloads>`__ the source code and
+If using PETSc < 3.14, `Download <https://bitbucket.org/petsc/petsc4py/downloads>`__ the source code and
 extract the correct version matching your PETSc version:
 
 .. prompt:: bash
 
    tar -xzf petsc4py-<version>.tar.gz
+   cd petsc4py-<version>
 
-From the ``petsc4py`` directory do a user-space install:
+From 3.14 onwards, petsc4py is included in the PETSc source code, in which case you can skip the above step and simply go straight to the petsc4py source directory:
 
 .. prompt:: bash
 
-   python setup.py install --user
+   cd $PETSC_DIR/src/binding/petsc4py
+
+Then do a user-space install:
```

Comment:
Not really a user-space install, I think we can just get rid of the user/system/environment stuff and assume users know what they're doing.

Reply:
> assume users know what they're doing.

Never a good idea

Reply:
Yeah I know. For those in the lab we have a pretty detailed page on setting up python, but for external users it's not really our job to explain how python works.